security/cmd/node_agent_k8s/nodeagent.yaml

```diff
@@ -15,3 +15,10 @@ spec:
       - name: nodeagent
         image: "{{ .Values.global.hub }}/node-agent-k8s:{{ .Values.global.tag }}"
         imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - mountPath: /tmp
```

Comment:
Suggest not to mount it at /tmp, what about something like /etc/sds ? Same below

Reply:
done(/var/run/sds) and verified